reference/upgrades.md

```diff
@@ -1,5 +1,92 @@
 ## Upgrading Mayastor
 
-Mayastor v2.0.0 does not support upgrade from previous versions. The installation of the new version should be preceded by the removal of any earlier version.
+Upgrade to the latest Mayastor version is supported only from 2.1.0 or later. To upgrade from a previous version, visit [Manual steps to upgrade Mayastor](manual-steps-to-upgrade-mayastor). 
```

Comment:
Upgrade feature is available from 2.1.0 release but hot-upgrade can be performed on version 2.0.0 / 2.0.1 version as well... so can we make it more simpler to write, what are the previous version to support, it seems some confusing statement.
cc: @avishnu

Reply:
I've added a section 'Supported upgrade paths'. This can be used for later releases too.

Reply:
I think this needs a bit of a re-phrase. Something along the lines:
Mayastor supports seamless upgrades starting with target version 2.1.0 and later, and source versions 2.0.0 and later.
To upgrade from a previous version (1.0.6 or prior) to 2.1.0 or later, refer the manual steps as documented here: <link>

Reply:
Sorry.. it should be 1.0.5 and prior

Reply:
Updated the text